src/components/Records/Search/Input/AdvancedSearch/AdvancedSearchDialogBox.vue

```diff
@@ -213,6 +213,19 @@ export default {
       }
     },
   },
+  mounted() {
+    //When the user is redirected to advancedsearch url directly
+    //it will open the dialog box
+    if (
+      this.$route.fullPath === "/advancedsearch" ||
+      this.$route.fullPath === "/advancedSearch"
```

Comment:
These two conditions look the same, unless I'm mistaken. Is this meant to be the case?

Reply:
Yes, this is meant for the case.